cmds/core/wget/wget_test.go

```diff
@@ -98,25 +99,20 @@ var tests = []struct {
 	},
 }
 
-func getFreePort(t *testing.T) int {
+func getFreePort(t *testing.T) (net.Listener, int) {
```

Comment:
nit: With the code as proposed, maybe it makes sense to rename this to "getListener" ?   We're not really returning a free port, we're returning a listener as well as the port it's listening on